test_vm/src/lib.rs

```diff
@@ -62,16 +63,18 @@ pub struct TestVM<'bs> {
     invocations: RefCell<Vec<InvocationTrace>>,
 }
 
-impl<'bs> TestVM<'bs> {
-    pub fn new(store: &'bs MemoryBlockstore) -> TestVM<'bs> {
+impl TestVM {
+    pub fn new(store: impl Into<Rc<MemoryBlockstore>>) -> TestVM {
+        let store = store.into();
         let mut actors =
-            Hamt::<&'bs MemoryBlockstore, ActorState, BytesKey, Sha256>::new_with_config(
-                store,
+            Hamt::<Rc<MemoryBlockstore>, ActorState, BytesKey, Sha256>::new_with_config(
+                Rc::clone(&store),
                 DEFAULT_HAMT_CONFIG,
             );
+
         TestVM {
             primitives: FakePrimitives {},
-            store,
+            store: store,
```

Comment:
```suggestion
            store,
```

Reply:
This is what clippy is complaining about.

Reply:
Done.